src/pages/Transaction/TransactionComp.tsx

```diff
@@ -123,42 +136,55 @@ export const TransactionOverview: FC<{ transaction: State.Transaction }> = ({ tr
   ]
   if (txStatus === 'committed') {
     if (confirmation >= 0) {
-      OverviewItems.push(
-        {
-          title: i18n.t('block.timestamp'),
-          content: parseSimpleDate(blockTimestamp),
-        },
-        bytes
-          ? {
-              title: `${i18n.t('transaction.transaction_fee')} | ${i18n.t('transaction.fee_rate')}`,
-              content: (
-                <div
-                  style={{
-                    display: 'flex',
-                  }}
-                >
-                  <DecimalCapacity value={localeNumberString(shannonToCkb(transactionFee))} />
-                  <span
+      if (isLite) {
```

Comment:
These code seems eccentric, because it seems appear twice.

Reply:
If you can not refactor the code instantly, please remain a TODO mark for it.

Reply:
updated by: d9501cb50e094c127487f312a5a6c6b2301c7abd